crates/cli/Cargo.toml

```diff
@@ -23,15 +23,17 @@ pacquet_registry     = { workspace = true }
 pacquet_tarball      = { workspace = true }
 pacquet_diagnostics  = { workspace = true }
 
-clap         = { workspace = true }
-futures-util = { workspace = true }
-rayon        = { workspace = true }
-reflink-copy = { workspace = true }
-reqwest      = { workspace = true }
-node-semver  = { workspace = true }
-serde        = { workspace = true }
-serde_json   = { workspace = true }
-tokio        = { workspace = true }
+async-recursion = { workspace = true }
```

Comment:
Are we using async-recursion? I couldn't find any reference.

Reply:
https://github.com/pnpm/pacquet/blob/9c94551927488ec8c1e87ec4688170d4541459a8/crates/cli/src/commands/install.rs#L48